src/main/java/com/github/manolo8/darkbot/config/Config.java

```diff
@@ -97,6 +97,7 @@ public static class Safety {
             public @Option("config.general.safety.revive_location") @Dropdown ReviveLocation REVIVE = ReviveLocation.BASE;
             public @Option @Visibility(Level.INTERMEDIATE) @Number(min = 5, max = 60, step = 10) int WAIT_BEFORE_REVIVE = 5;
             public @Option @Visibility(Level.INTERMEDIATE) @Number(min = 3, max = 15 * 60, step = 10) int WAIT_AFTER_REVIVE = 90;
+            public @Option @Visibility(Level.INTERMEDIATE) @Number(min = 0, max = Integer.MAX_VALUE, step = 10) int INSTANT_REPAIR = 0;
```

Comment:
I'd use a `@Number.Disabled` here, makes sense that this is a check-box style thing (with, additionally, a count).
Also, IMO avoid Integer.MAX_VALUE, that's a number users will rarely want to set, do 9999 or whatever seems like a reasonable number, that should be the maximum. Keep in mind the size of the spinner is set according to the maximum number (so a max-value will make a huge jspinner for no reason)